event-input.c

```diff
@@ -2324,7 +2335,7 @@ evin_iomon_keypress_cb(mce_io_mon_t *iomon, gpointer data, gsize bytes_read)
          * Additionally ignore all key events if proximity locked
          * during a call or alarm.
          */
-        if (((ev->code != KEY_CAMERA_FOCUS) &&
+        if ((ev->type == EV_KEY) && ((ev->code != KEY_CAMERA_FOCUS) &&
```

Comment:
This is already within "if type is key" block  & type is no longer changed -> no need to check it again -> undo changes